ibm_catalog.json

```diff
@@ -0,0 +1,574 @@
+            {
+              "key": "existing_boot_volume_kms_key_crn"
+            },
+            {
+              "key": "force_delete_kms_key"
+            },
+            {
+              "key": "use_boot_volume_key_as_default"
+            },
+            {
+              "key": "security_group"
+            },
+            {
+              "key": "security_group_ids"
+            },
+            {
+              "key": "boot_volume_key_name"
+            },
+            {
+              "key": "boot_volume_snapshot_id"
+            },
+            {
+              "key": "dedicated_host_id"
+            },
+            {
+              "key": "enable_floating_ip"
+            },
+            {
+              "key": "load_balancers"
+            },
+            {
+              "key": "manage_reserved_ips"
+            },
+            {
+              "key": "placement_group_id"
+            },
+            {
+              "key": "primary_virtual_network_interface_additional_ip_count"
+            },
+            {
+              "key": "secondary_allow_ip_spoofing"
+            },
+            {
+              "key": "secondary_floating_ips"
+            },
+            {
+              "key": "secondary_security_groups"
+            },
+            {
+              "key": "existing_secondary_subnet_id",
+              "custom_config": {
+                "type": "platform_resource",
+                "grouping": "deployment",
+                "original_grouping": "deployment",
+                "config_constraints": {
+                  "resourceType": "subnet",
+                  "selection": "single_select",
+                  "valueType": "id"
+                },
+                "associations": {
+                  "parameters": [
+                    {
+                      "name": "region",
+                      "optionsRefresh": true
+                    }
+                  ]
+                }
+              }
+            },
+            {
+              "key": "secondary_use_vsi_security_group"
+            },
+            {
+              "key": "snapshot_consistency_group_id"
+            },
+            {
+              "key": "use_static_boot_volume_name"
+            },
+            {
+              "key": "user_data",
+              "custom_config": {
+                "type": "textarea",
+                "grouping": "deployment",
+                "original_grouping": "deployment"
+              }
+            }
+          ],
+          "iam_permissions": [
+            {
+              "service_name": "is",
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::serviceRole:Manager",
+                "crn:v1:bluemix:public:iam::::role:Editor"
+              ]
+            },
+            {
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::serviceRole:Manager",
+                "crn:v1:bluemix:public:iam::::role:Editor"
+              ],
+              "service_name": "kms"
```

Comment:
Can we add notes to these permissions since its now supported (see internal [issue](https://github.ibm.com/ibmcloud/content-catalog/issues/5335)). The property name is `notes`. Your missing HPCS permissions which is supported too, but now you can use the notes feature to add a note with more info